load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -1379,19 +1411,83 @@ func (s *relaysSuite) querySharedParams(queryNodeRPCURL string) {
 	s.sharedParams = sharedParams
 }
 
+// queryAppParams queries the current on-chain application module parameters for use
+// over the duration of the test.
+func (s *relaysSuite) queryAppParams(queryNodeRPCURL string) {
+	s.Helper()
+
+	deps := depinject.Supply(s.txContext.GetClientCtx())
+
+	blockQueryClient, err := sdkclient.NewClientFromNode(queryNodeRPCURL)
+	require.NoError(s, err)
+	deps = depinject.Configs(deps, depinject.Supply(blockQueryClient))
+
+	appQueryclient, err := query.NewApplicationQuerier(deps)
+	require.NoError(s, err)
+
+	appParams, err := appQueryclient.GetParams(s.ctx)
```

Comment:
Why do all the other params need casting but this one doesn't?

Reply:
The `proofParams` need casting because its querier returns an interface [1]; which was added to avoid dependency cycles.

[1] https://github.com/pokt-network/poktroll/blob/main/pkg/client/interface.go#L335